@xen-orchestra/xapi/index.mjs

```diff
@@ -88,6 +88,20 @@ function onRetry(error) {
     })
   } catch (error) {}
 }
+function disconnectBeforeRetry(error) {
+  logErrorBeforeRetry.call(this, error)
+  const { arguments: args, this: self } = this
```

Comment:
Where is used arguments?

Reply:
`arguments` is not used, `args` is.

cf https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#assigning_to_new_variable_names_and_providing_default_values